main/tables.py

```diff
@@ -158,3 +158,18 @@ class Meta:
     def render_value(self, value: Decimal) -> str:
         """Render the value as a percentage."""
         return f"{value * 100:.0f}%"
+
+
+class MonthlyChargeTable(tables.Table):
+    """Table for the Monthly Charge listing."""
+
+    date = tables.DateColumn()
+    amount = tables.Column()
+    description = tables.Column()
+
+    class Meta:
+        """Meta class for the table."""
+
+        model = MonthlyCharge
+        fields = ("date", "amount", "description")
+        attrs: ClassVar[dict[str, str]] = {"class": "table table-striped"}
```

Comment:
If you indicate the fields of the model in the `Meta` section, you do not need to declare them manually.
```suggestion
    class Meta:
        """Meta class for the table."""

        model = MonthlyCharge
        fields = ("date", "amount", "description")
        attrs: ClassVar[dict[str, str]] = {"class": "table table-striped"}
```